api/src/main/java/org/apache/gravitino/credential/Credential.java

```diff
@@ -29,6 +29,9 @@ public interface Credential {
   /** Credential expire time in ms since the epoch. */
   String EXPIRE_TIME_IN_MS = "expire-time-in-ms";
 
+  /** The HTTP header used to get the credential from fileset location */
+  String HTTP_HEADER_CURRENT_LOCATION_NAME = "CurrentLocationName";
```

Comment:
The string constant defined in `Credential` are credential properties, it's odd to place `HTTP_HEADER_CURRENT_LOCATION_NAME` here, maybe you could place it to `CredentialConstants`

Reply:
updated